app.js

```diff
@@ -63,3 +63,14 @@ app.post('/driver/create', async (req, res) => {
     res.json();
   }
 });
+
```

Comment:
your route should go under the routes folder. create a file and or folder under the routes folder and move your code there.